zipline/utils/cache.py

```diff
@@ -334,7 +353,11 @@ def mkdir(self, *path_parts):
             The parts of the path after the working directory.
         """
         path = self.getpath(*path_parts)
-        os.mkdir(path)
+        try:
```

Comment:
Sure, I was more concerned with the fact that this will not fail if the directory already exists. Also, we have `paths.ensure_directory` which should do this.